<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>856</width>
    <height>650</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_7">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <widget class="QGroupBox" name="groupBox_4">
        <property name="minimumSize">
         <size>
          <width>500</width>
          <height>500</height>
         </size>
        </property>
        <property name="title">
         <string>显示</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <item>
          <widget class="QScrollArea" name="scrollArea">
           <property name="frameShape">
            <enum>QFrame::NoFrame</enum>
           </property>
           <property name="widgetResizable">
            <bool>true</bool>
           </property>
           <widget class="QWidget" name="scrollAreaWidgetContents">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>476</width>
              <height>461</height>
             </rect>
            </property>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_3">
        <property name="title">
         <string>测量</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QPushButton" name="openBtn2">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>打开图片</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="markBtn">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="text">
              <string>标点</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="measureBtn">
             <property name="text">
              <string>测量</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <item>
            <widget class="QLabel" name="label_10">
             <property name="text">
              <string>距离：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="distanceLE">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="unitCB">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <item>
              <property name="text">
               <string>mm</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>cm</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>m</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>411</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="title">
         <string>照相机标定</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>行角点数：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="rowNumSpB">
             <property name="minimum">
              <number>2</number>
             </property>
             <property name="maximum">
              <number>99</number>
             </property>
             <property name="value">
              <number>7</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>列角点数：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="colNumSpB">
             <property name="minimum">
              <number>2</number>
             </property>
             <property name="value">
              <number>7</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_8">
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>棋盘格边长</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="chessLenLE">
             <property name="text">
              <string>80</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_11">
             <property name="text">
              <string>mm</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QPushButton" name="openBtn1">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="text">
            <string>载入图片</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>已载入图片数量：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="imageNumLE">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QPushButton" name="startCalibBtn">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>开始标定</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="title">
         <string>光平面标定</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>光平面方程：</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QLineEdit" name="xCoeLE">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>70</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>70</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>x+</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="yCoeLE">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>70</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>70</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_7">
             <property name="text">
              <string>y+</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="zCoeLE">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>70</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>70</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_8">
             <property name="text">
              <string>z+</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="constCoeLE">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="maximumSize">
              <size>
               <width>70</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_9">
             <property name="text">
              <string>=0</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QPushButton" name="autoCreateBtn">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>自动生成光平面方程</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="customSetCB">
           <property name="text">
            <string>自定义光平面方程</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
